susemanager-utils/susemanager-sls/susemanager-sls.changes

```diff
@@ -1,4 +1,5 @@
-- util.systeminfo sls has been added to perform different actions at minion startup(bsc#1122381) 
```

Comment:
Next time, remember to add changes on top, so you don't get this problem :-)

Reply:
Ignore the comment. As you told me, this was just removing a trailing whitespace, that it's not clearly visible at this github diff :-)